import_notes.md

```diff
@@ -0,0 +1,66 @@
+# Initial Import Notes
+
+We will only use the word, additional readings and the definition(s)
+
+## Word itself
+
+The main reading for a word is the **first** keb element
+
+## Additional readings
+
+All keb elements after the first
+
+## Reading in kana
+
+reb element(s)
+
+## Definition
+
+gloss elements. Gloss elements are encased in "senses" which are groupings of
```

Comment:
Could you also mention that these "senses" are elements in the actual XML?
I was going crazy thinking it was linguistic term of some sort 😂